package.json

```diff
@@ -10,7 +10,9 @@
     "lint:css": "stylelint './**/src/**/*.(js|ts|jsx|tsx)'",
     "lint:js": "eslint .",
     "test": "jest",
-    "test:debug": "node --inspect-brk node_modules/.bin/jest --runInBand"
+    "test:cypress": "cd browser && cypress run",
+    "test:debug": "node --inspect-brk node_modules/.bin/jest --runInBand",
```

Comment:
Maybe calling this `test:jest` is more informative

Reply:
Actually... what's the purpose of `test:debug`?

Reply:
I added that the other day, it lets you use an interactive debugger with tests. See https://jestjs.io/docs/26.x/troubleshooting#tests-are-failing-and-you-dont-know-why

Reply:
Good question `test:debug`, upon brief research, appears to be a script using Jest's [recommended way](https://jestjs.io/docs/troubleshooting) of getting more info out of a failing test by running it in a Node process that allows use of `debugger` statements. 

As such, it's now been renamed `test:jest:debug`

Reply:
Also as long as we're doing this, perhaps we want a task something like `test:all` or `test:full` that runs Jest, Cypress, and whatever else we might later add?

Reply:
> Also as long as we're doing this, perhaps we want a task something like `test:all` or `test:full` that runs Jest, Cypress, and whatever else we might later add?

Good call, I've added `test:full` per your recommendation